FP_Factions/gordon_reskins/RHSak74/config.cpp

```diff
@@ -0,0 +1,115 @@
+class CfgPatches
+{
+	class gordon_reskins_RHSAK74stuff
+	{
+		// List of units defined in this "PBO"

```

Comment:
Comment not really needed is it? Sorta clear whats happening there tbh.

Reply:
Yeet

Reply:
Alright looking good. Will merge when I tested it.